packages/pn-pa-webapp/src/components/Notifications/NotificationPaymentPagoPa.tsx

```diff
@@ -66,9 +66,11 @@ const NotificationPaymentPagoPa: React.FC<Props> = ({ iun, payment }) => {
           </Typography>
         </Grid>
         <Grid item xs={12}>
-          <ButtonNaked color="primary" onClick={dowloadHandler}>
-            {t('payment.pagopa-notice')}
-          </ButtonNaked>
+          {payment.attachment && (
```

Comment:
I should put the same condition into dowloadHandler `!_.isNil(payment.recIndex) && payment.attachment`